src/subscription_manager/identity.py

```diff
@@ -169,3 +169,6 @@ def __str__(self):
         return "<%s, name=%s, uuid=%s, consumer=%s>" % \
                 (self.__class__.__name__,
                 self.name, self.uuid, self.consumer)
+
+    def log_str(self):
```

Comment:
__str__ at the moment isnt uniq, since it doesn't include a object id, so it shouldn't be a repr.

I'm not entirely sure what all uses Identity.__str__, so I figured I'd leave it alone. It can likely be
changed, just a little fuzzy what all uses auto-str'ing.